src/strategy/spid.ts

```diff
@@ -60,7 +60,7 @@ export class SpidStrategy extends SamlStrategy {
     super(options, verify);
     if (!options.requestIdExpirationPeriodMs) {
       // 1 hour
-      options.requestIdExpirationPeriodMs = 3600 * 1000;
+      options.requestIdExpirationPeriodMs = 3600 * 1000; // TODO: Change the default value or set it into io-backend?
```

Comment:
that would be better, probably keeping  a sane default here

Reply:
Fixed in 3198ea4